netkan/netkan/status.py

```diff
@@ -79,50 +81,64 @@ def export_all_mods(cls, compat: bool = True) -> Dict[str, Any]:
         return data
 
     @classmethod
-    def export_to_s3(cls, bucket: str, key: str, compat: bool = True) -> None:
+    def export_all_mods(cls, compat: bool = True) -> Dict[str, Any]:
+        data: Dict[str, Any] = {}
+        for game_id in ['ksp', 'ksp2']:
+            data.update({game_id: cls.export_game_mods(
+                condition=ModStatus.game_id == game_id, compat=compat)})
+        data.update({'no_game_id': cls.export_game_mods(
+            condition=ModStatus.game_id.does_not_exist(), compat=compat)})
+        return data
+
```

Comment:
```suggestion
```

On second thought, this isn't called anywhere. Maybe just delete it if we don't need it?

(Sorry for the notification spam, having a bit of a struggle with the GH web UI here)

Reply:
It's used by the dump_status utility, which will be involved in how the tables are transitioned from having no range key. Which is a destructive update (you can create secondary indexes, but I think for simplicity and the tiny amount of data, migrating makes sense).

All good, happy with the questions.

Reply:
Ahh, I see. From the way the diff looked I took this to be a new function, and ctrl-F didn't find it anywhere else.